### PR TITLE
Modify workflow to publish npm package

### PR DESCRIPTION
**Pull request by pruthvidhodda**
# Description

Modified workflow to publish npm package to npm registry

Edit: publish to github releases as well

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #41 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -31,11 +37,11 @@ jobs:
...
       if: startswith(github.ref, 'refs/tags/v')
-      run: echo "##[set-env name=DEPLOY_PACKAGE;]true"
+      run: echo "::set-env name=DEPLOY_PACKAGE::true"
       
     - name: Publish to npm
       if: env.DEPLOY_PACKAGE == 'true'
-      run: npm set registry https://registry.npmjs.org/ && npm publish
+      run: npm set registry https://registry.npmjs.org/ && npm publish --access public
       env:
         NODE_AUTH_TOKEN: ${{secrets.npm_token}}
```
Comment (youngbupark) [inline .github/workflows/build.yml:46]: should npm_token be `NPM_TOKEN` ?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -31,11 +37,11 @@ jobs:
...
```
Comment (youngbupark) [inline .github/workflows/build.yml:47]: the next step is publishing npm pkg to github. is it working properly?

File: .github/workflows/build.yml (hunk 1 above)
Comment (pruthvidhodda) [reply .github/workflows/build.yml]: Changed it

File: .github/workflows/build.yml (hunk 2 above)
Comment (pruthvidhodda) [reply .github/workflows/build.yml:47]: Just verified it on my fork: 

Release: https://github.com/pruthvidhodda/js-sdk/releases
CI on master: https://github.com/pruthvidhodda/js-sdk/runs/782197467

**Pull request merged**